resources/js/frontend/price.js

```diff
@@ -0,0 +1,122 @@
+jQuery(document).ready(function($) {
+    function adjustPrice(quantity, productId) {
+        $.ajax({
+            url: ajax_obj.ajax_url,
+            method: 'POST',
+            data: {
+                action: 'adjust_price_based_on_quantity',
+                quantity: quantity,
+                product_id: productId
+            },
+            success: function(response) {
+                if (response.success && response.data.new_price) {
+                    $('.price').html(response.data.new_price);
```

Comment:
does it work together with src/StoreKeeper/WooCommerce/B2C/templates/add-on/js/update-price.php ? What is one is also saving the price. 

Does it work with products on discount?

Reply:
yes, it works